browsers/firefox/manifest.json

```diff
@@ -78,7 +78,8 @@
         "storage",
         "<all_urls>",
         "alarms",
-        "identity"
+        "identity",
+        "identity.email"
```

Comment:
needed this as well as `identity`

Reply:
ah, i was wondering if we could reverse. just do the email one